### PR TITLE
add(main): new package jira-go

### PR DESCRIPTION
**Pull request by 2096779623**

### CODE REVIEW

File: packages/jira-go/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_make() {
+	termux_setup_golang
+	make
+}
+
+termux_step_make_install() {
+	install -Dm700 -t "${TERMUX_PREFIX}"/bin jira
+}
```
Comment (MrAdityaAlok) [inline packages/jira-go/build.sh:18]: Please add shell [completions](https://github.com/go-jira/jira#setting-up-tab-completion). It should be added in `postinst` script and removed in `prerm`.

File: packages/jira-go/build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+TERMUX_PKG_DESCRIPTION="Simple jira command line client in Go"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.0.27"
+TERMUX_PKG_SRCURL=https://github.com/go-jira/jira/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=c5bcf7b61300b67a8f4e42ab60e462204130c352050e8551b1c23ab2ecafefc7
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_make() {
```
Comment (MrAdityaAlok) [inline packages/jira-go/build.sh:11]: Remove this and instead call `termux_setup_golang` in `termux_step_pre_configure`.

Makefile will be recognised by build system.

**Review by MrAdityaAlok — CHANGES_REQUESTED** [review]

File: packages/jira-go/build.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	termux_setup_golang
+}
+
+termux_step_make_install() {
+	install -Dm700 -t "${TERMUX_PREFIX}"/bin jira
+}
+
+termux_step_create_debscripts() {
+	echo "#!$TERMUX_PREFIX/bin/sh" > postinst
+	echo "eval "$(jira --completion-script-bash)"" >> postinst
```
Comment (MrAdityaAlok) [inline packages/jira-go/build.sh:21]: You forgot to escape `$`. Or simply change outer quotes to (') single quotes.

Also:
- this command will print script to `stdout`. Redirect it to something like `$TERMUX_PREFIX/share/bash-completion/completions/jira_go`

- Zsh completions are also available. Just replace bash with zsh in that eval command.
File for `zsh`: `$TERMUX_PREFIX/share/zsh/site-functions/_jira_go`

File: packages/jira-go/build.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	chmod 0755 postinst
```
Comment (MrAdityaAlok) [inline packages/jira-go/build.sh:22]: You need not do this. It will be done automatically.

File: packages/jira-go/build.sh [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+	echo "#!$TERMUX_PREFIX/bin/sh" > postrm
+	echo "rm -rf $TERMUX_PREFIX/etc/bash_completion.d" >> postrm
```
Comment (MrAdityaAlok) [inline packages/jira-go/build.sh:24]: Change file names as requested in previous comments.

**Review by MrAdityaAlok — CHANGES_REQUESTED** [review]

File: packages/jira-go/build.sh (hunk 3 above)
Comment (2096779623) [reply packages/jira-go/build.sh]: Thanks!

Comment (MrAdityaAlok) [reply packages/jira-go/build.sh]: > Thanks!

You're welcome!

File: packages/jira-go/build.sh [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+	termux_setup_golang
+}
+
+termux_step_make_install() {
+	install -Dm700 -t "${TERMUX_PREFIX}"/bin jira
+}
+
+termux_step_create_debscripts() {
+	echo "#!$TERMUX_PREFIX/bin/sh" > postinst
+	echo 'eval "$(jira --completion-script-bash)"' >> postinst
```
Comment (MrAdityaAlok) [inline packages/jira-go/build.sh:21]: I think you misunderstood something. By redirecting I ment to do something like:
```bash
echo "mkdir -p $TERMUX_PREFIX/share/bash-completion/completions" > postinst
echo "jira --completion-script-bash > $TERMUX_PREFIX/share/bash-completion/completions/jira_go" > postinst
```

Similarly you can do for `zsh`.

File: packages/jira-go/build.sh [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+	echo 'eval "$(jira --completion-script-zsh)"' >> postinst
+	
+	echo "#!$TERMUX_PREFIX/bin/sh" > prerm
+	echo "rm -rf $TERMUX_PREFIX/etc/bash_completion.d" >> prerm
+	echo "rm -f $TERMUX_PREFIX/etc/profile.d/bash_completion.sh" >> prerm
```
Comment (MrAdityaAlok) [inline packages/jira-go/build.sh:26]: Remove line 25 and 26. It isn't needed.

**Review by MrAdityaAlok — APPROVED** [review]

**Pull request merged**